gui/masspit.lua

```diff
@@ -0,0 +1,218 @@
+-- Interface to assign caged units in a stockpile to pond/pit.
+
+local utils = require 'utils'
+local gui = require('gui')
+local widgets = require('gui.widgets')
+
+Masspit = defclass(Masspit, widgets.Window)
+Masspit.ATTRS {
+    frame_title='masspit',
+    frame={w=43, h=22},
+    resizable=true,
+    resize_min={w=43, h=12},
```

Comment:
you can omit specifying dimensions that are the same as the starting frame. `resize_min={h=12}` is equivalent. this helps when you're tweaking the frame size so you only have to change it in one place.